bindings/python/dlite-entity-python.i

```diff
@@ -39,6 +39,9 @@ class Metadata(Instance):
         return Instance.create_metadata(
             uri, dimensions, properties, description)
 
+    def __init__(self, *args, **kwargs):
+        pass  # do nothing, just avoid calling Instance.__init__()
```

Comment:
Why cannot the constructor of the subclass use the constructor of the superclass at all? To me, removing logic (i.e., functionality) means that the subclass is probably doing less than the superclass, which in turn implies that the class hierarchy might be technically incorrect. What is your thought about this?

Reply:
You have a point.

This is a simple pragmatic workaround for that SWIG does not considers subclassing of a SWIG-generated class.

This workaround avoids that the SWIG-internal function dlite.Instance.__init__() calls abort() because it gets a pointer to a dlite.Metadata instead of a dlite.Instance, which it expects.

Reply:
OK. You might want to consider to describe this SWIG behavior in a comment for this constructor (for future reference).

Reply:
Added!